templates/terraform/decoders/app_profile.erb

```diff
@@ -0,0 +1,11 @@
+if _, ok := res["multiClusterRoutingUseAny"]; ok {
+  res["multiClusterRoutingUseAny"] = true
+}
+
+if _, ok := res["name"]; ok {
```

Comment:
I've seen this pattern in a few places now. I think we should have a util function like `getImportIdQualifiers` but a little more general that extracts components of a `name` (or another) field into a map.